### ISSUE TITLE
examples for the charged small molecule

### ISSUE THREAD

**Issue by athar91** [ricalessandri/Martini3-small-molecules#7]
Please post some examples of the parameterization of the small molecules containing charged centers e.g., nitrogen containing heterocycles at physiological pH. or guidelines to treat such centers.

**Comment by ricalessandri** [comment]
Yep, some examples can be useful. I'll add this to my to-do list.

In the meantime, have a look at Supplementary Table 25 of the [Supp. Info. of the Martini 3 main paper](https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-021-01098-3/MediaObjects/41592_2021_1098_MOESM1_ESM.pdf). The table reports default bead assignments for cations and anions: these can be used as starting point to assign bead types to charged groups in small molecules.
